components/nimbus/src/targeting.rs

```diff
@@ -131,7 +124,27 @@ pub fn jexl_eval<Context: serde::Serialize>(
         })
         .with_transform("bucketSample", bucket_sample);
 
-    let res = evaluator.eval_in_context(expression_statement, context)?;
+    match evaluator.eval_in_context(expression_statement, context) {
+        Ok(v) => Ok(v),
+        Err(err) => Err(NimbusError::EvaluationError(err.to_string())),
+    }
```

Comment:
You want [`Result::map_err`](https://doc.rust-lang.org/stable/std/result/enum.Result.html#method.map_err)

```suggestion
    evaluator
        .eval_in_context(expression_statement, context)
        .map_err(|err| NimbusError::EvaluationError(err.to_string()))
```